go/engine/identify2_with_uid.go

```diff
@@ -343,23 +343,29 @@ func (e *Identify2WithUID) runIdentifyUI(ctx *Context) (err error) {
 	// fingerprints and the user's UID and username.
 	e.remotesReceived = e.them.BaseProofSet()
 
+	e.G().Log.Debug("| IdentifyUI.Start(%s)", e.them.GetName())
```

Comment:
This can't happen due to the single flight lock around the whole thing.

On Tuesday, June 21, 2016, Patrick Crosby <notifications@github.com> wrote:

> In go/engine/identify2_with_uid.go
> <https://github.com/keybase/client/pull/3149#discussion_r67879098>:
>
> > @@ -343,23 +343,29 @@ func (e *Identify2WithUID) runIdentifyUI(ctx *Context) (err error) {
> >  	// fingerprints and the user's UID and username.
> >  	e.remotesReceived = e.them.BaseProofSet()
> >
> > +	e.G().Log.Debug("| IdentifyUI.Start(%s)", e.them.GetName())
>
> Would there be any value in also adding a string that would identify each
> runIdentifyUI call? In case there are multiple overlapping calls for the
> same username. Something like:
>
> | IdentifyUI.Start(max) [abcd]
> | IdenitfyUI.ReportLastTrack(max) [xyz0]
> | IdentifyUI.ReportLastTrack(max) [abcd]
>
> etc.
>
> ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/keybase/client/pull/3149/files/b72be0aa2e7f7e66af822653df1890bed89be9e9#r67879098>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AA05_xVpbUc-cq23hy8xQkqfmevGtKTyks5qN_VmgaJpZM4I6rRX>
> .
>